devops/llvm-anybuild.yml

```diff
@@ -218,12 +224,16 @@ jobs:
   
   - script: |
       set -euo pipefail
-      rm -f $(PKG_NAME).tar.gz
-      tar zcf $(PKG_NAME).tar.gz build/install
-      md5sum $(PKG_NAME).tar.gz | awk '{print $1}' > $(PKG_NAME).tar.gz.md5
+      rm -f $(PKG_NAME).tar
+      tar --sort=name --owner=root:0 --group=root:0 --mtime='UTC 2021-01-01' -cf $(PKG_NAME).tar build/install
```

Comment:
I think the problem was the zip part that you had before, which is what creates the time stamp. We're not zipping anymore because it doesn't shrink enough to be worth the extra minutes, so we don't have that problem.

If you want to zip, there seems to be two [alternatives](https://serverfault.com/questions/110208/different-md5sums-for-same-tar-contents): use an external gzip or pass `-n` to the internal one.

Reply:
Oh well, we are zipping! :astonished: 

I remember looking at it and thinking I should fix this, but apparently, I didn't. 

Honestly, this hasn't been a problem at all for us, so I think we shouldn't change the time on the file just for that.

Just upload the file and its MD5 and let's see how it works.

Reply:
removed --mtime, --owner, and --group